samples/clearResultsAndQueues.py

```diff
@@ -22,7 +22,16 @@
     print "Clearing all queues"
     for queue in queues:
         storage_service_queue = QueueService(account_name = config.storage_account_name, sas_token = queue[1])
-        storage_service_queue.clear_messages(queue[0])
+        print "\tClearing " + queue[0]
+        while True:
```

Comment:
Possibility for an infinite loop here? What if a poison message can't be removed for some reason? Maybe too much of an edge case, but wanted to double check that we shouldn't break out of the catch.

Reply:
This is part of the setup, so I am not super concerned about this edge case.